Themes/_fallback/Languages/sk.ini

```diff
@@ -0,0 +1,3049 @@
+Quads=Quady
+Quantize=Kvantovať
+RandomBackgroundMode=Režim
+RandomModifiers=Modifikátory
+Rate=Rýchlosť
+Record in selection=Rekord vo výbere
+RefreshRate=Obnovovacia frekvencia
+Reload Songs=Znovu načítať skladby/maratóny
+Remove=Odstrániť
+RemoveCombinations=Odstrániť
+RemoveFeatures=
+Remove Change=Odstrániť zmenu
+Remove Keysound=Odstrániť zvuk klávesu
+Rename=Premenovať
+Reset To Defaults=Vrátiť na predvolené hodnoty
+Revert from disk=Vrátiť sa k verzii na disku
+Revert to last save=Vrátiť sa k verzii vo vyr. pamäti
+Rolls=Spŕšky
+Save=Uložiť
+SaveScores=Ukladať skóre
+SaveReplays=Ukladať záznamy
+SaveStats=Ukladať štatistiky
+ScoreDisplay=Zobrazenie skóre
+Scoreboard=Tabuľka so skóre
+ScoringType=Typ skórovania
+ScreenFilter=Filter obrazovky
+Scroll=Posúvanie
+Scroll segments=Posúvanie častí
+Secret=Tajné
+Secs Remaining=Ostáva sekúnd
+Server=Server
+Servers=Servery
+Set BG Fit Mode=Nastaviť režim roztiahnutia
+Set P1=Nastaviť P1
+Set P2=Nastaviť P2
+Set modifiers=Nastaviť modifikátory
+Set Mods=Nastaviť Mody
+Set selection start=Nastaviť začiatok výberu
+Set selection end=Nastaviť koniec výberu
+Shift all timing changes down=Posunúť všetky zmeny časovania nižšie
+Shift all timing changes up=Posunúť všetky zmeny časovania vyššie
+Shift timing in region down=Posunúť časovanie v oblasti nižšie
+Shift timing in region up=Posunúť časovanie v oblasti vyššie
+ShowBanners=Zobrazovať nadpisy
+ShowBeginnerHelper=Pomocník pre začiatočníkov
+ShowCaution=Upozornenie
+ShowDancingCharacters=Tanečníci
+ShowDanger=Nebezpečenstvo
+ShowInstructions=Inštrukcie
+ShowLyrics=Text
+ShowNativeLanguage=Prepis
+ShowSelectGroup=Skupina skladieb
+ShowSongOptions=Zobraziť nastavenia skladby
+ShowStats=Zobrazovať štatistiky
+SmoothLines=Vyhladenie čiar
+SoundEffect=Zvukový efekt
+Speed segments=Segmenty rýchlosti
+Starting Time=Čas začiatku
+StepMania Credits=Zásluhy a poďakovania
+Stops=Pauzy
+Center1Player=1&nbsp;hráč na stred
+Song=Skladba
+Song Group=Skupina skladieb
+Song Options=Možnosti skladby
+SongsPerPlay=Skladieb na hru
+Sort=Triediť
+SoundResampleQuality=Kvalita prevzorkovania
+SoundVolume=Hlasitosť zvuku v hre
+SoundVolumeAttract=Hlasitosť zvuku v režime demo
+Speed=Rýchlosť
+Speed Increment=Kroky rýchlosti
+Speed Increment Large=Veľké kroky rýchlosti
+Step Author=Autor krokov
+Steps=Kroky
+Stream=Stream
+StretchBackgrounds=Roztiahnuť pozadia
+Style=Štýl
+Sub title=Vedľajší názov
+Sub title transliteration=Prepis vedľajšieho názvu
+SuperGrowth=Super rast
+Tap Steps=Normálne kroky
+Tempo=Tempo
+Test Input=Otestovať vstup
+Test Lights=Otestovať svetlá
+TextureColorDepth=Farba textúr
+Theme=Motív
+ThreeKeyNavigation=Pohyb v ponukách
+Tickcount=Tickcount
+TimingWindowScale=Náročnosť poroty
+Time Signatures=Taktové predznamenania
+Transform=Transformovať
+Transfer Edits to USB=Preniesť úpravy na USB
+Transfer Edits from USB=Preniesť úpravy na USB
+Type=Typ
+Turn=Otočenie
+Undo=Vrátiť
+UseUnlockSystem=Odomknúť systém
+View steps data=View Steps Data
```

Comment:
Still in english

Reply:
Thank you. I think it would be ideal to consult this with a group of experienced Slovak players, do you have an idea how to contact them?

Reply:
Actually not aware of any slovak palyers :(

Reply:
Ok, let's try to go with Slovak and if someone complains about it we can go back to English (or alternatively: Slovak (English))